metricbeat/module/windows/perfmon/pdh_integration_windows_test.go

```diff
@@ -233,7 +238,11 @@ func TestLongOutputFormat(t *testing.T) {
 	}
 	defer query.Close()
 	counter := CounterConfig{Format: "long"}
-	err = query.AddCounter(processorTimeCounter, counter, false)
+	path, err := query.GetCounterPaths(processorTimeCounter)
+	if err != nil {
+		t.Fatal(err)
+	}
+	err = query.AddCounter(path[0], counter, false)
```

Comment:
Do we need to check if len(path) > 0 before using path[0]?

Reply:
It will always trip on err!= nil if there are any cases where the path is empty. Else, it should have the original counter path. I have added the check either way.

Reply:
ahh sorry I didn't realize this is in the test code...